wgpu-core/src/pipeline.rs

```diff
@@ -164,6 +164,12 @@ pub struct ProgrammableStageDescriptor<'a> {
     /// The name of the entry point in the compiled shader. There must be a function with this name
     /// in the shader.
     pub entry_point: Cow<'a, str>,
+    /// Specifies the values of pipeline-overridable constants in the shader module module.
+    ///
+    /// The key represents the numeric ID of the constant, if one is specified, and otherwise the constant's identifier name.
+    ///
+    /// The value may represent any of WGSL's concrete scalar types.
+    pub constants: Cow<'a, naga::back::PipelineConstants>,
```

Comment:
question: Is `pipeline_constants` overly verbose in this context? I'm not too familiar with how a “programmable stage” and a pipeline are linked, so I might not realize that it should be fine to elide the “pipeline” context.

This question applies to other places where this identifer is used for a new field in like manner.